lib/src/components/NxSegmentedButton/NxSegmentedButton.scss

```diff
@@ -43,8 +43,8 @@
   // lock the line-height here so that it is not affected by the font-size increase on the icon
   line-height: 1em;
   margin: 0;
-  padding-left: var(--nx-spacing-3x);
-  padding-right: var(--nx-spacing-3x);
+  padding-left: calc(var(--nx-spacing-3x) - 1px);
+  padding-right: calc(var(--nx-spacing-3x) - 1px);
```

Comment:
Why do we need the `-1 px`?

Reply:
> Why do we need the `-1 px`? I think we can get rid all of the addition and subtraction?

Oh the `-1px` is so border is inclusive as part of the padding.

Reply:
> Oh the -1px is so border is inclusive as part of the padding.

This is on the dropdown arrow part of the button.  Checking sketch closely, the arrow itself isn't even the same size in the implementation vs the mockup so that needs to be fixed before we can really know what's supposed to be done here.

Reply:
> This is on the dropdown arrow part of the button. Checking sketch closely, the arrow itself isn't even the same size in the implementation vs the mockup so that needs to be fixed before we can really know what's supposed to be done here.

That said I do suspect at least the right-side `-1` is correct here.  The style it's overridding also uses `-1` and it makes sense with the border.  The inner distance (left padding) though, I'm not sure about.

Reply:
> > This is on the dropdown arrow part of the button. Checking sketch closely, the arrow itself isn't even the same size in the implementation vs the mockup so that needs to be fixed before we can really know what's supposed to be done here.
> 
> That said I do suspect at least the right-side `-1` is correct here. The style it's overridding also uses `-1` and it makes sense with the border. The inner distance (left padding) though, I'm not sure about.

I updated the icon size here: 307103d22bc47cf81b89614255a161921bf83652

The total width is now 36 (12px padding with the borders + 12px icon) compared to 38px (13px padding with the borders + 12px icon) in the mockup.
I think the mockup might not be updated as intended: the border is not inclusive as part of the horizontal padding but the vertical padding is 🤔. I'll consult with the designers!

Reply:
> I think the mockup might not be updated as intended: the border width is not inclusive as part of the horizontal padding but the vertical padding is thinking. I'll consult with the designers!

The mockups are a little odd in regards to the divider line between the two parts of the button.  For one thing, the spacing appears to be different in primary buttons vs secondary!  But on secondary buttons it appears that he spacing to right of the divider is a standard distance while the spacing to the left of the divider is one less.  Which seems inconsistent to me.

Reply:
> > I think the mockup might not be updated as intended: the border width is not inclusive as part of the horizontal padding but the vertical padding is thinking. I'll consult with the designers!
> 
> The mockups are a little odd in regards to the divider line between the two parts of the button. For one thing, the spacing appears to be different in primary buttons vs secondary! But on secondary buttons it appears that he spacing to right of the divider is a standard distance while the spacing to the left of the divider is one less. Which seems inconsistent to me.

I've consulted with Jason with this and they updated the Sketch file so it's all consistent.